src/engines/misc/sends/sends.cpp

```diff
@@ -25,22 +29,79 @@ namespace otto::engines::sends {
     OTTO_UNREACHABLE;
   }
 
-  Sends::Sends() : audio(std::make_unique<Audio>()), screen_(std::make_unique<Screen>()), props{{*audio, *screen_}} {}
+  Sends::Sends(core::ui::Icon i) : audio(std::make_unique<Audio>()), screen_(std::make_unique<Screen>(i)), props{{*audio, *screen_}}
+  {
+    props.dry.on_change().connect([&](float a) {
+      if (is_recursive) return;
+      set(a, props.stored_levels.dry, props.stored_levels.fx1, props.stored_levels.fx2);
+    });
+    props.fx1.on_change().connect([&](float a) {
+      if (is_recursive) return;
+      set(a, props.stored_levels.fx1, props.stored_levels.dry, props.stored_levels.fx2);
+    });
+    props.fx2.on_change().connect([&](float a) {
+      if (is_recursive) return;
+      set(a, props.stored_levels.fx2, props.stored_levels.dry, props.stored_levels.fx1);
```

Comment:
You can be fairly proud of the solution to this problem - it looks nice in the end!

Reply:
thanks